safe-global/safe-wallet-web
# [Counterfactual] Safe creation

## What it solves

Resolves #3179 

## How this PR fixes it

- Adjust safe creation flow to create counterfactual safes for 1/1 setups
- Adds fallbacks for CGW calls so the UI is unblocked
- Fetches native balance
- Adds feature flag
- Uses an incremental saltNonce for all safe deployments

## How to test it

1. Connect your wallet and create a 1/1 safe
2. Observe no cost on the review screen
3. Press Create
4. The dashboard should be immediately visible
5. There should be no errors
6. Send ETH to the safe address
7. Observe it being visible on the assets page
8. Create another 1/1 safe
9. Observe the same safe address

## Screenshots

## Checklist
